build.gradle

```diff
@@ -25,6 +30,7 @@ apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'org.ajoberstar.grgit'
+apply plugin: 'com.adarshr.test-logger'
```

Comment:
@andrii-kovalenko-ct Maybe consider using gradle's newer plugin mechanism, where you only:

````groovy
buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "com.adarshr:gradle-test-logger-plugin:1.1.2"
  }
}

apply plugin: "com.adarshr.test-logger"
````
[Why it's better?](https://plugins.gradle.org/plugin/com.adarshr.test-logger#new-plugin-mechanism-info-body)

Reply:
i honestly don't like new plugins format, because they split `id` and `version`: i like old oneliner style more. Also, new plugins mechanism is still incubating. If they make it stable - maybe we will re-factor the script.

Reply:
actually the oneliner is the new mechanism :) the one with no splitting :)

Reply:
ok, i mean not one line, but solid line:
_com.adarshr:gradle-test-logger-plugin:1.1.2_
instead of
_id "com.adarshr.test-logger" version "1.1.2"_